source/tutorials/nixos-configuration-on-vm.md

```diff
@@ -127,25 +124,26 @@ The complete `configuration.nix` file now looks like this:
 
 A virtual machine is created with the `nix-build` command.
 
-To select `configuration.nix` in the working directory, specify the configuration file as an argument:[^nixosrebuild]
-
-[^nixosrebuild]: On NixOS you can create a virtual machine using `nixos-rebuild build-vm -I nixos-config=./configuration.nix`, which wraps the original command.
+To select `configuration.nix` in the working directory, specify the configuration file as an argument:
 
 ```shell-session
-nix-build '<nixpkgs/nixos>' -A vm -I nixos-config=./configuration.nix
+nix-build '<nixpkgs/nixos>' -A vm \
+-I nixpkgs=channel:nixos-22.11 \
+-I nixos-config=./configuration.nix
 ```
 
-:::{note}
-On systems that do not have a `$NIX_PATH` environment variable set up as recommended in [the pinning tutorial](pinning-nixpkgs) you use:
+This command builds the attribute `vm` using the version of Nixpkgs as using the [short form for channels]((https://nixos.org/manual/nix/stable/command-ref/opt-common.html)) and using the NixOS configuration as specified in the relative path.
 
+
+:::{note}
+It is common to a `$NIX_PATH` environment variable set up so you can use your current version of nixpkgs to build the virtual machine:[^nixosrebuild]
```

Comment:
Now that this degraded to an option, and we shouldn't leave options in tutorials (remember, they are cognitive overhead for learners under cognitive load), and we discourage `NIX_PATH` anyway, we may as well leave it out entirely.